spec/components/banner_component_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "rails_helper"
+
+RSpec.describe BannerComponent, type: :component do
+  context "with left align image" do
+    before do
+      render_inline(described_class.new(
+        title: "Banner title",
+        text: "Some random text",
+        image: {
+          url: "media/images/test/image.png",
+          alt: ""
+        },
+        background_color: "purple",
+        link: "http://www.example.com"
```

Comment:
No tests here to actually assert that the title, text, and link are being output. Seems worth having a quick one to check the values passed in are being respected.